resources/views/components/game/leaderboards-listing/leaderboards-list-item.blade.php

```diff
@@ -17,16 +17,13 @@ class="leading-3"
 
         <p>{{ $leaderboard->description }}</p>
 
-        @php
-            $bestScore = $leaderboard->sortedEntries()->first();
-        @endphp
-        @if (!$bestScore)
+        @if (!$leaderboard->topEntry)
             <p>No entries.</p>
         @else
             <div class="flex justify-between">
-                {!! userAvatar($bestScore->user, iconSize: 16, iconClass: 'rounded-sm') !!}
+                {!! userAvatar($leaderboard->topEntry->user->username, iconSize: 16, iconClass: 'rounded-sm') !!}
```

Comment:
You can pass the User model directly to userAvatar to avoid requerying it.

Reply:
I keep forgetting that is a thing. Fixed in latest.